HICPlugin/HICPlugin.csproj

```diff
@@ -34,6 +34,9 @@
   <ItemGroup>
     <None Include="Resources\MTo1.png" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Drawing.Common" Version="8.0.6" />

```

Comment:
Where did this one sneak in from...?

Reply:
Looks like it's needed with the move to net8  https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/system-drawing-common-windows-only  not sure show this was working up till now...

Reply:
If it's needed, it will be needed within RDMP.UI not within HICPLUGIN